repo/packages/K/kubernetes/3/config.json

```diff
@@ -99,6 +99,11 @@
           "maximum": 65535,
           "type": "integer"
         },
+        "apiserver-runtime-config": {
+          "default": "",
+          "description": "A set of key=value pairs that describe runtime configuration that may be passed to apiserver. apis/<groupVersion> key can be used to turn on/off specific api versions. apis/<groupVersion>/<resource> can be used to turn on/off specific resources. api/all and api/legacy are special keys to control all and legacy api versions respectively.",
+          "type": "string"
```

Comment:
if it's straightforward, it might be nice to provide a `pattern` validator for this new config item